packages/rum-core/src/error-logging/error-logging.js

```diff
@@ -171,22 +171,24 @@ class ErrorLogging {
   }
 
   logPromiseEvent(promiseRejectionEvent) {
-    const prefix = 'Unhandled promise rejection: '
-    const { reason } = promiseRejectionEvent
+    const prefix = 'Uncaught (in promise) '
+    const { reason = '<no reason specified>' } = promiseRejectionEvent
+    let errorEvent
 
-    if (reason == null) {
-      this.logError(prefix + '<no reason specified>')
-    } else if (typeof reason.message === 'string') {
+    if (typeof reason.name === 'string' && typeof reason.message === 'string') {
```

Comment:
name shouldn't be a required field here. It should work for error like objects as well. I would do this

```js
const name = reason.name ? reason.name + ':' : ''

this.logErrorEvent({
  error: reason,
  message: `${prefix}${name} ${reason.message}`
})
```

Reply:
By doing this, it would also work correctly for `Promise.reject({message: 'testerror'})`